snap/snapcraft.yaml

```diff
@@ -192,38 +176,122 @@ parts:
     after:
       - snapcraft-desktop-helpers
     plugin: nil
-    override-stage: |
+    override-build: |
+      snapcraftctl build
       rm -f $SNAPCRAFT_PART_INSTALL/usr/share/applications/python3.6.desktop
-      snapcraftctl stage
     stage-packages:
       - firefox
 
   gtk3-apps:
     plugin: nil
     stage-packages:
-      - gnome-sudoku
-      - gnome-mahjongg
       - gnome-chess
       - gnuchess
       - gnuchess-book
 
   native-apps:
     plugin: nil
+    override-build: |
+      mkdir -p $SNAPCRAFT_PART_INSTALL/usr/share/applications/
+      cat > $SNAPCRAFT_PART_INSTALL/usr/share/applications/glmark2-wayland.desktop <<EOF
+      [Desktop Entry]
+      Name=GLMark2 (Wayland)
+      Exec=glmark2-es2-wayland
+      Type=Application
+      Icon=ubuntu-logo
+      Categories=Utility
+      EOF
+      cat > $SNAPCRAFT_PART_INSTALL/usr/share/applications/glmark2-x11.desktop <<EOF
+      [Desktop Entry]
+      Name=GLMark2 (X11)
+      Exec=glmark2-es2
+      Type=Application
+      Icon=ubuntu-logo
+      Categories=Utility
+      EOF
```

Comment:
Wouldn't it be cleaner to put these in `.desktop` files in `glue/`? Is that a thing you can easily do?

Reply:
Actually, there's probably too much in "glue" already. I should clean that up.

These probably should go in a glmark2 directory tree for clarity and to avoid them being orphaned in a future rework.

Can we land this version and I'll follow up with a fuller rework PR?

Reply:
Sure